apps/offlineevents/templates/meinberlin_offlineevents/offlineevent_detail.html

```diff
@@ -8,6 +8,18 @@
 
             <div class="item-detail">
 
+                <div class="phase-info">
+                    <h3 class="phase-info__title">
+                        {% trans 'Offline-Event' %}
+                    </h3>
+                    <div class="phase-info__subtitle">
+                        {{ object.date|date:'DATE_FORMAT' }}
+                    </div>
+                    <div class="phase-info__description">
+                        {% trans 'Here you can read the documentation on what happened during this Offline-Event.' %}
```

Comment:
I guess we have to distinguish whether the event is in the past or future and display different texts.